php/libraries/Utility.class.inc

```diff
@@ -539,7 +539,7 @@ class Utility
      */
     static function toArray($var)
     {
-        if (!isset($var[0])) {
+        if (!array_key_exists(0, $var)) {
```

Comment:
@jstirling91 what if the element you are passing in an array but not at the correct level... ie:

you want: 
```
array(
          [0] => array(
                              [name]=>"rida", 
                              [quality]=>"the best"
                     )
)
```

but you get:
```
array(
         [name]=>"rida", 
         [quality]=>"the best"
)
```